app/src/main/java/ru/vssemikoz/newsfeed/MainApplication.java

```diff
@@ -0,0 +1,41 @@
+package ru.vssemikoz.newsfeed;
+
+import android.app.Application;
+import android.content.Context;
+
+import androidx.room.Room;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+import ru.vssemikoz.newsfeed.DataBase.NewsAppDataBase;
+import ru.vssemikoz.newsfeed.api.NewsApi;
+
+public class MainApplication extends Application {
```

Comment:
Good job! What we need to do is to override `onCreate()` and move all initialization there.